contract/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=================================
+Contracts for recurrent invoicing
+=================================
+
+ * This module forward-port to v9 the contracts management with recurring
+   invoicing functions.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Sales -> Contracts and select or create a new contract.
+#. Check *Generate recurring invoices automatically*.
+#. Fill fields and add new lines. You have the possibility to use markers in
+   in description field to show init date and end of invoiced period.
```

Comment:
s/to show init date and end of invoiced period/to show the start and end date of the invoiced period